### PR TITLE
Moyo Home work

### PR DESCRIPTION
**Pull request by Ostafii4uk**
- [DEMO LINK](https://Ostafii4uk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Ostafii4uk.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+a {
```
Comment (solaryasha) [inline src/style.css:5]: Avoid styling tags

File: src/style.css [hunk 2]
```diff
@@ -1 +1,66 @@
...
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  padding: 23px 0;
+  position: relative;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
```
Comment (solaryasha) [inline src/style.css:42]: Better to define font's in `html` tag and inherit in links

**Review by solaryasha — APPROVED** [review]

File: src/style.css (hunk 1 above)
Comment (Ostafii4uk) [reply src/style.css:5]: Only selectors, I understood)

**Comment by Ostafii4uk** [comment]
Aproved

**Pull request closed without merging**